data.json

```diff
@@ -232,5 +232,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "SimbaChain",
+    "difficulty": "easy",
+    "description": "Register and get a free Blockchain Shirt!",
```

Comment:
It would be great to add a mention about the block chain controversy (eth_listtransactions) => look at the issue for references

Reply:
just a comment about the the controversy?

Reply:
Look at the second link of the issue